sentry/src/middleware/channel.rs

```diff
@@ -0,0 +1,43 @@
+use crate::db::field::{BigNumPg, ChannelIdPg};
+use crate::db::DbPool;
+use bb8_postgres::tokio_postgres::{types::Json, Row};
+use hyper::{Body, Request};
+use primitives::channel::ChannelId;
+use primitives::{Channel, ChannelSpec};
+use std::error;
+
+pub async fn channel_load(
+    mut req: Request<Body>,
+    (pool, id): (DbPool, &ChannelId),
+) -> Result<Request<Body>, Box<dyn error::Error>> {
+    let channel_id = hex::encode(id);
+
+    let channel = pool
```

Comment:
We should probably move this to more general-purpose place, instead of leaving it here so we can reuse it.
I am opened for suggestions, but this is **not** part of this PR.